web/src/lib/components/forms/api-key-form.svelte

```diff
@@ -11,7 +11,7 @@
   export let title = 'API Key';
   export let cancelText = 'Cancel';
   export let submitText = 'Save';
-  export let apiKeyName = 'API Key';
+  let apiKeyName = apiKey.name ?? 'API Key';
```

Comment:
When is the apiKey name null ?

Reply:
Empty != null

Right now, it won't be used at all, so no need to add it. IMO that's a backend logic